src/main/java/com/tabnine/inline/InlineCompletionChangeHandler.java

```diff
@@ -0,0 +1,12 @@
+package com.tabnine.inline;
+
+import com.intellij.codeInsight.CodeInsightActionHandler;
+import com.intellij.openapi.editor.Editor;
+import com.intellij.openapi.project.Project;
+import com.intellij.psi.PsiFile;
+import org.jetbrains.annotations.NotNull;
+
+public class InlineCompletionChangeHandler implements CodeInsightActionHandler {
```

Comment:
why is this class exists with empty implementation?

Reply:
Good catch. Deleted.